packages/asc-ui/src/components/Select/SelectStyle.ts

```diff
@@ -63,7 +64,7 @@ export const SelectIcon = styled(ChevronDown)`
 const SelectStyle = styled.select<Props>`
   width: 100%;
   height: 100%;
-  padding: ${themeSpacing(0, 3)};
+  padding: ${themeSpacing(2, 3)};
```

Comment:
What is the reason for this extra padding?

Reply:
This aligns the padding with the input field